lua/terrortown/entities/items/item_ttt_disguiser.lua

```diff
@@ -8,18 +8,26 @@ if SERVER then
 	AddCSLuaFile()
 end
 
-ITEM.EquipMenuData = {
-	type = "item_active",
-	name = "item_disg",
-	desc = "item_disg_desc"
-}
-ITEM.material = "vgui/ttt/icon_disguise"
 ITEM.CanBuy = {ROLE_TRAITOR}
 ITEM.oldId = EQUIP_DISGUISE or 4
 
 if CLIENT then
 	local trans
 
+	ITEM.EquipMenuData = {
+		type = "item_active",
+		name = "item_disg",
+		desc = "item_disg_desc"
+	}
+	ITEM.material = "vgui/ttt/icon_disguise"
+	ITEM.hud = Material("vgui/ttt/perks/hud_disguiser.png")
```

Comment:
Does the disguiser not have a description?

Reply:
Yes, I just moved this part of the code a few lines down. There is no new description

Reply:
I missed the fallback handling nvm this